public_html/behavioralhealth/estacada/resources.html

```diff
@@ -130,345 +130,578 @@ <h1 class="blue no_margin some_margin" style="font-size: 40px;">Behavioral Healt
                                 <ul style="padding-left: 40px">
                                     <li><a href="#" data-scrollto="#resources">Community Resources</a>
```

Comment:
remove this out bullet point for community resources and the one for support groupls. We want the new list to be the first level of bullet points